src/ItemsCarousel/PlaceholderCarousel.js

```diff
@@ -44,13 +44,15 @@ export class PlaceholderCarousel extends React.Component {
     const {
       containerWidth,
     } = this.state;
-
+    let self = this;
     return (
       <Measure
         bounds
         margin={false}
         onResize={({ bounds }) => {
-          this.setState({ containerWidth: bounds.width });
+          requestAnimationFrame(function () {
+            self.setState({ containerWidth: bounds.width });
```

Comment:
Maybe use an arrow function and remove the self variable ?
```
requestAnimationFrame(() => this.setState({ containerWidth: bounds.width }));
```

Reply:
@bitriddler this is fixed